helm/kong-app/templates/_helpers.tpl

```diff
@@ -25,13 +25,22 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{- define "kong.imagesHash" -}}
+{{- if .Values.image.registry }}
+{{- printf "%s-%s-%s" .Values.image.registry (include "kong.getRepoTag" .Values.image) (include "kong.getRepoTag" .Values.ingressController.image) | sha256sum | trunc 63 -}}
+{{- else }}
```

Comment:
Same as above?
```suggestion
{{- else -}}
```

Reply:
Tried locally, does not make any difference.